### ISSUE TITLE
Search result return type missing some important information

### ISSUE THREAD

**Issue by SimonCommits** [lca1/medco-connector#126]
Dear all,

It would be nice to fetch more information from the database using the function get_ontology_elements.
For the moment the current return type of this function is the following:
```
CREATE OR REPLACE FUNCTION medco_ont.get_ontology_elements(search_string varchar, lim integer DEFAULT 10)
      RETURNS TABLE (
        c_fullname varchar,
        c_name varchar,
        c_visualattributes char(3),
        c_basecode varchar,
        c_metadataxml text,
        c_comment text,
        m_applied_path varchar,
        id int,
        fullpath text)
```

I think we should make the function return at least the same attributes returned by the http query returned by browsing the ontology normally. In my opinion we should be able to use the search result in the same that we can use the elements returned by browsing the ontology.
The current use case I see is when dropping the search result in a dropzone I implemented. After dropping the concept in the dropzone a check verifies if the `value_type` of the concept dropped is indeed of numerical type.

Cheers.

**Comment by f-marino** [comment]
@WiseIndian what are the fields that are missing? Because I am returning exacting the same object in all the search APIs, i.e. `exploreSearchResultElement`. If there are fields missing, these are probably populated in the frontend. I know @jimmydjabali had the same problems as you in the beginning, but he managed to fix it, maybe can give you some hints.

**Comment by SimonCommits** [comment]
Hello @f-marino you can check the difference between the fields returned by the search query and a query that is a result from browsing the ontology.

The search query returns the following
![actual](https://user-images.githubusercontent.com/8803839/144471627-b8d19e7a-3aab-455e-ac5f-8e1223e3f424.png)

Browsing the ontology returns the following 
![expected](https://user-images.githubusercontent.com/8803839/144471902-60ec1453-8b8f-4eee-b0d9-6d862570ac39.png)

You can see that the search feature returns an empty metadata content. The metadata field contains the information DataType which is used to determine what is the value of the value type field of a tree node. Hence the default value SIMPLE is given to this tree node field by means of functions that pre-existed the search feature instead of NUMERICAL which is inferred from the PosFloat value of the JSON. This is an exemple field that is missing and for which I need the information for the bioref project. But you can see that other information than the DataType are missing from the JSON returned from the server. And this might cause unexpected issues at a later time.

**Comment by f-marino** [comment]
Ok so the problem is not the sql function, which is correctly returning the xml metadata, is the connector that is not sending the metadata to glowing bear. There was a bug in the unmarshalling, [this PR](https://github.com/ldsec/medco/pull/127) should fix it. While debugging I noticed an error in the synthetic SPO dataset, there are two Unit Values elements in the xml metadata, while there should be only one. I modified the definition of the metadataxml in the api to avoid umarshalling errors for the moment, but this should be fixed in the dataset.
